lib/services/navigator_app.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:marvel/model/character.dart';
+import 'package:marvel/screens/second_page.dart';
+import 'package:marvel/screens/third_page.dart';
+import 'package:marvel/services/app_routes.dart';
+import 'package:marvel/screens/home_page.dart';
+
+class NavigatorApp extends StatelessWidget {
+  const NavigatorApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Navigator(
+      onGenerateRoute: generateRoute,
+    );
+  }
+
+  Route<dynamic>? generateRoute(RouteSettings settings) {
```

Comment:
Please extract this in separate file. 
In programming we have a Single responsibility principle which is the first from the SOLID principles - please read a bit about it.

Reply:
Done